expertsystem/ui/__init__.py

```diff
@@ -263,6 +263,10 @@ def convert_edges_in_graph(
                 convert_edges_in_graph(instance)  # type: ignore
             elif isinstance(instance[0], tuple):
                 convert_edges_in_graph([g for g, _ in instance])  # type: ignore
+        elif isinstance(instance, dict):
+            graph_settings_groups = instance
+            for graphs in graph_settings_groups.values():
+                convert_edges_in_graph([g for g, _ in graphs])
```

Comment:
@spflueger While this addition seems trivial, I have my doubts about the implementation, because I believe this conversion over the `values` of this
```python
Dict[Strength, List[Tuple[StateTransitionGraph[ParticleWithSpin], Dict[int, InteractionNodeSettings]]]]
```
(namely, the `graph_settings_groups`) creates a new `list` of graphs for each strength.

Reply:
Still, even if this is true, this code is temporary and should be moved out asap once #254 has been resolved.

Reply:
yeah the follow up PR to this will already remove that conversion